app/src/main/java/eu/kanade/presentation/reader/settings/GeneralSettingsPage.kt

```diff
@@ -73,4 +73,9 @@ internal fun ColumnScope.GeneralPage(screenModel: ReaderSettingsScreenModel) {
         label = stringResource(MR.strings.pref_flash_page),
         pref = screenModel.preferences.flashOnPageChange(),
     )
+
+    CheckboxItem(
+        label = stringResource(MR.strings.pref_disable_zoom_out),
+        pref = screenModel.preferences.disableZoomOut(),
+    )
```

Comment:
Should be in [ReadingModePage.kt](https://github.com/mihonapp/mihon/blob/6bdb37be65757ca903c5c2a249ca03331b04d673/app/src/main/java/eu/kanade/presentation/reader/settings/ReadingModePage.kt) under `WebtoonViewerSettings`